### ISSUE TITLE
Hang when connecting using "hardware mouse" option without mouse connected

### ISSUE THREAD

**Issue by iskarian** [mariotaku/moonlight-tv#439]
### Moonlight Version

1.6.18

### Device Type

webOS TV

### Device Model

LG C3

### Device OS

8.3.1

### Host OS

None

### Host Software

None

### Bug Description

I have a LG C3, and have a combo keyboard/mouse connected to it via bluetooth. I am able to see the host computer and pair with it. When I have the "hardware mouse" option enabled, but the mouse is not connected, the app hangs at "Connecting" indefinitely. The host logs show no activity when this happens, so I suspect the hang is occurring before the client even attempts to connect to the host.

### Reproduction Steps

1. Enable "hardware mouse"
2. Ensure no mouse is connected
3. Attempt to connect to the host
4. App hangs at "Connecting"

### Expected Behavior

App connects successfully. (When I have the "hardware mouse" option disabled, or when I have the mouse connected, I am able to connect successfully.)

**Comment by mariotaku** [comment]
This was an known issue, and has been fixed in [v1.6.16](https://github.com/mariotaku/moonlight-tv/releases/tag/v1.6.16). Please install the latest version and see if the issue still exists.

**Comment by iskarian** [comment]
My apologies, I mistyped. I'm on 1.6.**18**. (I only installed it two days ago, after all!) I took a quick look at the diff; it looks like there's only a check for whether the config option is set, not whether there's an actual mouse to wait for? 

https://github.com/mariotaku/moonlight-tv/blob/2cad8b3a375c19249d43803a29fe74466b6117b0/src/app/stream/session_worker.c#L25C1-L27C1

...and then `session_evmouse_wait_ready` spins until `mouse->dev` is no longer `NULL`, but of course if no mouse is found, `mouse->dev` remains `NULL` forever.
